docs/changes.rst

```diff
@@ -225,6 +225,14 @@ Enhancements
   and to `tocimxmlstr()` to allow control over whether the object is
   interpreted as a value or as a declaration. (Issue #950).
 
+* Added a new conversion function: `cimtype()` takes a CIM data typed value
```

Comment:
shouldn't we say Added a new conversion function to the public API?

Reply:
That cements what we are doing in the first sentence.  Applies to the following also.

Reply:
DONE. Both changed as proposed.